src/cljs/s3_beam/client.cljs

```diff
@@ -144,7 +164,8 @@
                        defaults to read-string.
     :key-fn          - a function used to generate the object key for the uploaded file on S3
                        defaults to nil, which means it will use the passed filename as the object key.
-    :headers-fn      - a function used to create the headers for the GET request to the signing server."
+    :headers-fn      - a function used to create the headers for the GET request to the signing server.
+    :progress-events - if true, progress events will be pushed on to channel during upload."
```

Comment:
I wonder if this should maybe be `:progress-events?` since it's a boolean flag? Or do we only do that with functions that return boolean values?